crypto/ffc/ffc_params.c

```diff
@@ -15,7 +15,7 @@
 
 void ffc_params_init(FFC_PARAMS *params)
 {
-    memset(params, 0, sizeof(FFC_PARAMS));
+    memset(params, 0, sizeof(*params));
```

Comment:
Hmm that's the second time I did that....

Reply:
It won't be the last for either of us.